dev/dartdoc.dart

```diff
@@ -56,6 +56,22 @@ dependencies:
     args.add(name.substring(0, name.length - 5));
   }
 
+  [
```

Comment:
Can I assume that we're running from `.` so the relative path of `bin/cache/...` exists?